From felipe.balbi@nokia.com  Thu Apr 29 12:37:19 2010
From: Felipe Balbi <felipe.balbi@nokia.com>
Date: Thu, 25 Mar 2010 13:25:24 +0200
Subject: usb: musb: omap2430: remove unused define
To: Greg KH <greg@kroah.com>
Message-ID: <1269516328-28267-8-git-send-email-felipe.balbi@nokia.com>


From: Anand Gadiyar <gadiyar@ti.com>

get_cpu_rev() is unused in this driver. It is probably legacy
code. So remove it.

Signed-off-by: Anand Gadiyar <gadiyar@ti.com>
Signed-off-by: Felipe Balbi <felipe.balbi@nokia.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/musb/omap2430.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/usb/musb/omap2430.c
+++ b/drivers/usb/musb/omap2430.c
@@ -39,10 +39,6 @@
 #include "musb_core.h"
 #include "omap2430.h"
 
-#ifdef CONFIG_ARCH_OMAP3430
-#define	get_cpu_rev()	2
-#endif
-
 
 static struct timer_list musb_idle_timer;
 
